### PR TITLE
Updated M2 with latest Funcotator info.

### LINKED ISSUE #5253: Add excluded fields parameter to funcotator.wdl
### Tool(s) or class(es) involved
Funcotator

### Description
For funcotator.wdl, please add the excluded field parameter as was done in #5242 for mutect2.wdl and mutect2_nio.wdl.

### PR DESCRIPTION
**Pull request by jonn-smith**
Now the Mutect2 WDL has all the hooks to run Funcotator (`Funcotator.wdl` was updated as well).

The `Funcotate` task in the M2 WDL is identical to that same task in `Funcotator.wdl`.

Fixes #5253

### CODE REVIEW

File: docs/funcotator/forum_info/forum_post_tutorial.md [hunk 1]
```diff
@@ -112,19 +114,73 @@ Versioned gzip archives of data source files are provided here:
 
 <a name="1.1.2.2"></a>
 ### 1.1.2.2 - gnomAD
-The pre-packaged data sources include gnomAD, a large database of known variants.  gnomAD is split into two parts - one based on exome data, one based on whole genome data.
+The pre-packaged data sources include a subset gnomAD, a large database of known variants.  This subset contains a set of INFO fields, primarily containing allele frequency data.  gnomAD is split into two parts - one based on exome data, one based on whole genome data.  These two data sources are not equivalent and for complete coverage using gnomAD, we recommend annotating with both.
```
Comment (davidbenjamin) [inline docs/funcotator/forum_info/forum_post_tutorial.md]: subset *of* gnomAD

Comment (davidbenjamin) [reply docs/funcotator/forum_info/forum_post_tutorial.md]: Specify that the subset of INFO fields is much smaller than the original vcf.  It is, right?

File: scripts/mutect2_wdl/mutect2.wdl [hunk 2]
```diff
@@ -52,12 +52,17 @@
 ##
 ## Funcotator parameters (see Funcotator help for more details).
 ## funco_reference_version: "hg19" for hg19 or b37.  "hg38" for hg38.  Default: "hg19"
-## funco_transcript_selection_list: Transcripts (one GENCODE ID per line) to give priority during selection process.
-## funco_transcript_selection_mode: How to select transcripts in Funcotator.  ALL, CANONICAL, or BEST_EFFECT
+## funco_output_format: "MAF" to produce a MAF file, "VCF" to procude a VCF file.  Default: "MAF"
+## funco_compress: (Only valid if funco_output_format == "VCF" )  If true, will compress the output of Funcotator.  If false, produces an uncompressed output file.  Default: false
+## funco_use_gnomad_AF: If true, wil include gnomAD allele frequency annotations in output by connecting to the internet to query gnomAD (this impacts performance).  If false, will not annotate with gnomAD.  Default: false
```
Comment (davidbenjamin) [inline scripts/mutect2_wdl/mutect2.wdl]: will include with an extra 'l' in "will"

File: docs/funcotator/forum_info/forum_post_tutorial.md (hunk 1 above)
Comment (jonn-smith) [reply docs/funcotator/forum_info/forum_post_tutorial.md]: Good catch! 

Yes it's the info fields that were reduced in size.

**Pull request merged**